apps/web/src/pages/Teams/IndexTeamPage/components/TeamTable/helpers.tsx

```diff
@@ -4,7 +4,11 @@ import { IntlShape } from "react-intl";
 import { Link, Chip } from "@gc-digital-talent/ui";
 import { commonMessages, getLocalizedName } from "@gc-digital-talent/i18n";
 import { notEmpty } from "@gc-digital-talent/helpers";
-import { Maybe, RoleAssignment, Team } from "@gc-digital-talent/graphql";
+import {
+  Maybe,
+  RoleAssignment,
+  TeamTable_TeamFragment as TeamTableTeamFragmentType,
+} from "@gc-digital-talent/graphql";
```

Comment:
Alias to avoid collision with the query defined

https://github.com/GCTC-NTGC/gc-digital-talent/blob/55edc65ef045f82252d584b5434e460ec70eff12/apps/web/src/pages/Teams/IndexTeamPage/components/TeamTable/TeamTable.tsx#L35-L37